ui/js/controllers/perf/graphs.js

```diff
@@ -841,6 +841,7 @@ perf.filter('testNameContainsWords', function() {
         var filters = textFilter.split(/\s+/);
         return _.filter(tests, function(test) {
             return _.every(filters, function(filter) {
+                test.name = test.name.toLowerCase();
                 return test.name.indexOf(filter) !== -1;
```

Comment:
We don't want to actually change the test.name property permanently, which is what this change does. Instead, try something like:

```js
return test.name.toLowerCase().indexOf(filter) !== -1;
```